pandas/core/groupby/generic.py

```diff
@@ -104,12 +104,19 @@ def _cython_agg_blocks(self, how, alt=None, numeric_only=True,
 
                 obj = self.obj[data.items[locs]]
                 s = groupby(obj, self.grouper)
-                result = s.aggregate(lambda x: alt(x, axis=self.axis))
+                try:
```

Comment:
What's this for? Not immediately obvious the link between this and the overall PR

Reply:
this handles blocks that return NotImplementedError and then cann't be aggregated, e.g. Categoricals with string categories, aggregating with mean for exampel (and ``numeric_only=False``) is passed

Reply:
Can you give an example? I'm also having trouble seeing this.

Reply:
If those return NotImplementedError can we limit the scope of the catching to just that?